resources/js/Components/MessageBox.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <Message :type="type">
+        <span>
+            <slot />
+        </span>
+    </Message>
+</template>
+
+<script lang="ts">
+    import { Message } from '@wmde/wikit-vue-components';
+
+    import defineComponent from '../types/defineComponent';
+
+    export default defineComponent({
+        props: {
+            type: String
+        },
+        components: {
+            Message
+        }
+    });
+</script>
+
+<style lang="scss">
+@import '~@wmde/wikit-tokens/dist/_variables.scss';
+
+.wikit.wikit-Message {
+    max-width: 705px;
```

Comment:
This is probably a bit too arbitrary here, and should be kept at the context of the page (to size the containing section rather than the message itself).

Reply:
Ok, I removed the MessageBox component and put styles for both the box radius and the message/intro sections width into `app.scss` now.